exonum-java-binding/core/rust/ejb-app/start_cryptocurrency_node.sh

```diff
@@ -41,6 +41,7 @@ CORE_TXT="core/target/ejb-core-classpath.txt"
 CRYPTOCURRENCY_TXT="cryptocurrency-demo/target/cryptocurrency-classpath.txt"
 EJB_CLASSPATH="$(cat ${EJB_ROOT}/${CORE_TXT}):$(cat ${EJB_ROOT}/${CRYPTOCURRENCY_TXT})"
 EJB_CLASSPATH="${EJB_CLASSPATH}:${EJB_ROOT}/core/target/classes"
+#TODO: remove when service loader implemented https://jira.bf.local/browse/ECR-2953
```

Comment:
let's use ticket id only, without the full link

Reply:
done